Cabal/Distribution/Simple/GHC/ImplInfo.hs

```diff
@@ -65,20 +66,26 @@ ghcVersionImplInfo ver = GhcImplInfo
   , flagProfAuto         = v >= [7,4]
   , flagPackageConf      = v <  [7,5]
   , flagDebugInfo        = v >= [7,10]
+  , supportsPkgEnvFiles  = v >= [8,0,2] -- broken in 8.0.1, fixed in 8.0.2
```

Comment:
@dcoutts if we know the snapshot date when it was fixed, we could say e.g.

```hs
 v >= [8,0,1,20161002]
```

That's the the current date-stamped version of my ghc-8.0.2 ppa snapshot:

```
$ /opt/ghc/8.0.2/bin/ghc --numeric-version
8.0.1.20161002

$ /opt/ghc/8.0.2/bin/ghc --info | grep Git
 ,("Project Git commit id","a24092ff501028ca1245b508320493f394378495")

```